www/public/repl/repl.js

```diff
@@ -165,11 +165,37 @@ async function processInputQueue() {
 // Callbacks to JS from Rust
 // ----------------------------------------------------------------------------
 
+var ROC_PANIC_INFO = null;
+
+function send_panic_msg_to_js(rocstr_ptr, tag_id) {
```

Comment:
Hmm this was called alignment earlier but tag_id here. Only one of those names can be accurate!
Do you know which?
We should maybe fix the Rust code in `test_gen` too, if it has the same confusion.

Reply:
it's the panic tag (indicates what kind of panic it is). I've tried to make it consistent everywhere now

Reply:
Great 👍